_pkgdown.yml

```diff
@@ -41,6 +41,10 @@ reference:
  - contents:
    - has_keyword('utils_fil')
 
```

Comment:
do you think it better we group these all as admiralonco utilities? 

has_keyword('utils_admiralonco')

Reply:
i followed admiral core convention here to separate these out on the Reference page. if we end up not having so many for admiralonco though i'd be happy to re-visit later and bundle these altogether if it makes things neater